src/mqt/bench/benchmarks/ae.py

```diff
@@ -11,8 +11,18 @@
 from __future__ import annotations
 
 import numpy as np
+from packaging.version import Version
+from qiskit import version
 from qiskit.circuit import QuantumCircuit
-from qiskit.circuit.library import PhaseEstimation
+
+if Version(version.get_version_info()) >= Version("1.3.2"):
+    from qiskit.circuit.library import phase_estimation
+else:
+    from qiskit.circuit.library import PhaseEstimation
```

Comment:
I'd slightly prefer the Pythonic "try and except" pattern that does not require us to import packaging as well as the `version` module from qiskit.
This, and all the other similar statements, can simply become
```suggestion
try
    from qiskit.circuit.library import phase_estimation
except ImportError:
    from qiskit.circuit.library import PhaseEstimation
```

Reply:
This leads to `TypeError: 'module' object is not callable` when using Qiskit `v1.2.1`, probably because the file is called `phase_esimation.py`?

Reply:
For the except block, should we go with:

```
from qiskit.circuit.library import PhaseEstimation

    def phase_estimation(num_eval: int, grover: QuantumCircuit) -> PhaseEstimation:
        """PhaseEstimation (Qiskit < 1.3.2)."""
        return PhaseEstimation(num_eval, grover)
```
        
or just ignore the `N813 Camelcase` warning?

`from qiskit.circuit.library import PhaseEstimation as phase_estimation  # noqa: N813`